context/app/api/vitessce.py

```diff
@@ -320,8 +334,36 @@ def _build_vitessce_conf(self):
             found_images = _get_matches(file_paths_found, files[0]["rel_path"])
             # Do not show IMS images that are in the "/separate/" folder.
             no_ims_separate_images = _exclude_matches(found_images, r"/separate/")
-            layer = self._build_multi_file_image_layer_conf(no_ims_separate_images)
-            conf["layers"] = [layer]
+            # File paths are like path/to/HybCycle_N/MMStack_PosM.ome.tif and we want
+            # each conf to be all images across a given PosM, each image named HybCycle_N.
+            # This is unique to seqFish.
+            if "seqFish" in self.entity["data_types"]:
+                is_valid_directory = all([re.match(SEQFISH_PATH, file) for file in found_images])
+                if not is_valid_directory:
+                    raise NameError("Directory structure for seqFish is incorrect")
```

Comment:
Thanks! If we ever actually hit this, in the process of debugging will probably want to add more information to error message: Might collect just those paths which are not valid, and include the RE, so the user could start to understand what went wrong without going back to the code.

```suggestion
                    raise Exception("Directory structure for seqFish is incorrect")
```
I would only raise [Exception](https://docs.python.org/3/library/exceptions.html#Exception), or a subclass we make. I don't think we want to re-use the other built in exceptions with our own semantics.